### PR TITLE
Revised for Ruby 2.4.0

### PR DESCRIPTION
**Pull request by CheezItMan**
## Description
I ran through installfest with Sierra.

## Todos
Are any of these items still in progress?
- Nope

## Feedback Requested
Please utilize the built-in GitHub Reviewers functionality to assign necessary reviewers. Use this section to further specify portions of the PR that you would like additional or focused feedback on.

### CODE REVIEW

File: 00-programming-fundamentals/03-installfest.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Xcode is an **integrated development environment** (often called an **IDE**) for
 ### Trust but Verify
 Run `$ xcode-select -p`. If it tells you `/Library/Developer/CommandLineTools`, you're ready to move on.
 
+If the Xcode Command-Line tools are not installed you can install it with:
+`xcode-select -p` and be patient because it will take time to install.
```
Comment (Hamled) [inline 00-programming-fundamentals/03-installfest.md]: The paragraph above this mentions running `xcode-select -p` already.

Will students need to run the command twice to install the command-line tools? If so we might want to make that more explicit.

Comment (CheezItMan) [inline 00-programming-fundamentals/03-installfest.md]: Doh!  Somehow there was a duplicate line.

File: 00-programming-fundamentals/03-installfest.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ Xcode is an **integrated development environment** (often called an **IDE**) for
 **You should have installed Xcode Command-Line tools in JumpStart.**
 
 ### Trust but Verify
-Run `$ xcode-select -p`. If it tells you `/Library/Developer/CommandLineTools`, you're ready to move on.
+
+If the Xcode Command-Line tools are not installed you can install it with:
+`xcode-select -p` and be patient because it will take time to install.
```
Comment (Hamled) [inline 00-programming-fundamentals/03-installfest.md]: Apologies, it looks like I was incorrect with my earlier comment.

Students need to do the following:
1. Run `xcode-select -p` to display the current path of the installed Xcode command line tools.
1. IF the previous command displayed a directory path, then they can move on to the Homebrew instructions.
1. IF the previous command displayed something like: `xcode-select: error: unable to get active developer directory, use xcode-select --switch to set one (or see man xcode-select)` then they should run the command `xcode-select --install` before moving on.

**Comment by Hamled** [comment]
Thanks for going through all of this again, looks good!

**Pull request merged**